ansi/progress.go

```diff
@@ -0,0 +1,39 @@
+package ansi
+
+import "strconv"
+
+// ResetProgress is a sequence that resets the progress bar to its default
+// state (hidden).
+//
+// OSC 9 ; 4 ; 0 BEL
+//
+// See: https://learn.microsoft.com/en-us/windows/terminal/tutorials/progress-bar-sequences
+const ResetProgress = "\x1b]9;4;0\x07"
+
+// SetProgress returns a sequence for setting the progress bar to a specific
+// percentage (0-100) in the "default" state.
+//
+// OSC 9 ; 4 ; 1 ; Percentage BEL
+//
+// See: https://learn.microsoft.com/en-us/windows/terminal/tutorials/progress-bar-sequences
+func SetProgress(percentage int) string {
```

Comment:
shouldn't it report to the fd? could be the best name would be `SetProgressReport` or something like that since return a string